examples/srd_sn/README.md

```diff
@@ -0,0 +1,43 @@
+The structure of input dataset follows the same pattern as
```

Comment:
I will propose a few clarifications in a follow-up commit.

Reply:
Okay!

Reply:
I incorporated your edits. Additionally cleared a sentence describing the nature of Hubble Diagram inputs: unbinned or redshift binned.